install/troubleshooting/column_stats_helper.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: 'Resolving the "Limited access to table column statistics" warning'
+backlink_href: /docs/install/troubleshooting
+backlink_title: 'Installation Troubleshooting'
+---
+
+import { MonitoringUserColumnStats } from "../../components/MonitoringUserSetupInstructions";
+
+You may get the following warning when running a collector test:
+
+```
+Warning: Limited access to table column statistics detected in database postgres. Please set up the monitoring helper function pganalyze.get_column_stats (https://github.com/pganalyze/collector#setting-up-a-restricted-monitoring-user) or connect as superuser, to get column statistics for all tables.
+```
+
+This warning indicates that column statistics for the specific database are not being reported by the collector.
+Column statistics are used by pganalyze to improve Index Advisor recommendations. Index Advisor will still work
+without the helper functions, but index recommendations may be less accurate.
+
+To resolve this, find the name for the database role that's configured as the monitoring user
+in your collector installation (we'll assume "pganalyze" here).
+
+<MonitoringUserColumnStats username="pganalyze" />
```

Comment:
Nice!

Reply:
Yeah, it was nice to reuse this. I had to get a little creative with the copy since the first paragraph in this component starts with "Then, ...", but I think it works.